views/includes/inputs/date-input.html

```diff
@@ -0,0 +1,34 @@
+{{ govukDateInput({
```

Comment:
I would move this date-input.html file to view/includes/inputs/date folder so it is in same location as the other date 'includes'

Reply:
ok, happy to leave as it is